kili/mutations/asset/__init__.py

```diff
@@ -249,3 +252,38 @@ def generate_variables(batch):
                                               generate_variables,
                                               GQL_DELETE_MANY_FROM_DATASET)
         return format_result('data', results[0], Asset)
+
+    @Compatible(['v1', 'v2'])
+    @typechecked
+    def add_assets_to_review(
```

Comment:
Why not use the dedicated resolver in backend `addAllLabeledAssetsToReview` that ensures the assets have a label of type default and their status is Labeled ?

Reply:
@p-desaintchamas I saw that resolver but with its name I thought that it was appending all labeled assets to review. this is not the case ?